osu.Game.Tests/Visual/SongSelect/TestSceneBeatmapMetadataDisplay.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Utils;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Screens.Play;
+using osuTK;
+
+namespace osu.Game.Tests.Visual.SongSelect
+{
+    [System.ComponentModel.Description("player loader beatmap metadata")]
```

Comment:
i don't think this description is very useful

Reply:
I've used this because `BeatmapMetadataDisplay` sounded a bit off, but I guess this test scene should be considered irrelevant from any mentioning of player loaders, will remove.

Reply:
that, and the name of the subject under test is in the test class name. there is no need to repeat information.